docker/kanister-redis/image/Dockerfile

```diff
@@ -0,0 +1,14 @@
+ARG TOOLS_IMAGE
+FROM ${TOOLS_IMAGE} AS TOOLS_IMAGE
+
+FROM debian:bullseye
+COPY --from=TOOLS_IMAGE /usr/local/bin/restic /usr/local/bin/restic
+COPY --from=TOOLS_IMAGE /usr/local/bin/kopia /usr/local/bin/kopia
```

Comment:
Do we need restic and kopia binary to be present in the tooling app?

Reply:
fixed